Makefile

```diff
@@ -32,7 +32,7 @@ operator-sdk:
 golangci-lint:
 	@type -p golangci-lint > /dev/null || GOFLAGS="" GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
 
-build:
+build: lint		# Reject unlinted code in CI.
```

Comment:
Can we re-evaluate the set of linters and the golangci-lint version we use in a follow-up PR please? I tend to believe that we are using really old linters.

Reply:
Sure, open a JIRA or just start a PR with proposal.